content/boards/pro-micro/index.md

```
@@ -19,6 +19,22 @@
 - 12 digital IO pins, 5 with PWM support.
 - 9 analog inputs (can be used as digital inputs).
 
+
```

Markdownlint (0.37.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)